packages/scandipwa/src/component/ProductAttributes/ProductAttributes.component.js

```diff
@@ -26,6 +26,38 @@ export class ProductAttributes extends PureComponent {
         attributesWithValues: AttributeType.isRequired
     };
 
+    renderGroups() {
+        const { attributesWithValues } = this.props;
+
+        const groups = Object.entries(attributesWithValues).map(
+            // eslint-disable-next-line no-unused-vars
+            ([attributeLabel, valueLabel]) => ({
```

Comment:
If you don't need object key in a map, you can use `Object.values` instead of `Object.entries`

Reply:
Updated in last PR.